go/example_integration_test.go

```diff
@@ -0,0 +1,139 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestDremio(t *testing.T) {
```

Comment:
@howareyouman I was gonna delete this but it did help me catch a bug which so it might be useful to have it as an integration. But I agree this is very flaky. What do think? Should we delete it or leave it?

Reply:
It's better to remove it, from the design perspective that in JUnit tests you are comparing the logs.
We should verify that we are rewriting all the tests from this one to the mocks and then remove this test file.